deployer/README.md

```diff
@@ -264,6 +268,12 @@ This sub-command can be used to generate the resource allocation choices for giv
 ##### `generate resource-allocation choices`
 This generates a custom number of resource allocation choices for a certain instance type, depending on a certain chosen strategy that can be used in the profile list of a hub.
 
+##### `generate resource-allocation daemonset-overhead`
+Updates `daemonset_overhead.yaml` with an individual cluster's DaemonSets with running pods combined requests of CPU and memory, excluding GPU related DaemonSets.
```

Comment:
This is currently a bit hard to understand. Maybe expand it a bit more with info from the PR's top comment?

```suggestion
This script updates information about a specific cluster in the `daemonset_overhead.yaml` file.

This file contains information about cpu/memory requests made by
DaemonSets with running pods in _all_ of our clusters but excluding GPU related DaemonSets.
```

Reply:
The command will not collect information from each cluster. I considered doing that, but I don't think the complexity added to `deployer` is justified when we can easily invoke it for all clusters one at the time with a command like `ls config/clusters | xargs -I {} deployer generate resource-allocation daemonset-overhead {}`.

I'll mention that in the docs here!